test/index_test.js

```diff
@@ -3,88 +3,97 @@
  * Module dependencies.
  */
 
-import { default as isValidItin, mask } from '../src';
 import should from 'should';
+import { isValid, mask } from '../src';
 
 /**
- * ITIN sample numbers.
+ * `Individual Taxpayer Identification Number` samples.
  */
 
-const invalidNumbers = [
-  '00070000',
-  '10070000',
-  '20070000',
-  '30070000',
-  '40070000',
-  '50070000',
-  '60070000',
-  '70070000',
-  '80070000',
-  '90000000',
-  '90010000',
-  '90020000',
-  '90030000',
-  '90040000',
-  '90050000',
-  '90060000',
-  '90089000',
-  '90089000',
-  '90093000'
-];
-const validNumbers = [
-  '900700000',
-  '900709999',
-  '999709999',
-  '900880000',
-  '900889999',
-  '999889999',
-  '900900000',
-  '900909999',
-  '999909999',
-  '900920000',
-  '900929999',
-  '999929999',
-  '900940000',
-  '900949999',
-  '999949999',
-  '900990000',
-  '900999999',
-  '999999999'
-];
+const numbers = {
+  invalid: ['00070000', '100_70_000', '20070000f', '300#70#000'],
+  valid: {
+    format: ['900-70-0000', '900 70 9999', '999-70 9999', '900 88 0000'],
```

Comment:
We've discussed this before but after looking at moment's isValid/strict option, I think we should consider making `strict` validate the numbers + format and delimiters instead. TBD.